src/components/Card.js

```diff
@@ -1,14 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './Card.css';
 
-const Card = () => {
-    return (
-        <div className="Card">
-            <h3>Caption</h3>
-            <hr/>
-            <p>Some text here...</p>
-        </div>
-    );
+class Card extends Component {
```

Comment:
Ideally each component should be placed in a separate folder - along with its css file. Then reference to it in App.js file will looks like _'./components/Card/Card'_ which is not quite good-looking. This could be resolved by using best-practices of naming conventions in style-guide I've provided recently:
![image](https://user-images.githubusercontent.com/52723192/92129878-c1671d00-ee0c-11ea-8699-d739ba91dd7c.png)